cpp/src/arrow/type.cc

```diff
@@ -716,6 +740,111 @@ std::vector<std::string> Schema::field_names() const {
   return names;
 }
 
+SchemaBuilder::SchemaBuilder(ConflictPolicy policy) : policy_(policy) {}
+
+SchemaBuilder::SchemaBuilder(std::vector<std::shared_ptr<Field>> fields,
+                             ConflictPolicy policy)
+    : fields_(std::move(fields)),
+      name_to_index_(CreateNameToIndexMap(fields_)),
+      policy_(policy) {}
+
+SchemaBuilder::SchemaBuilder(const std::shared_ptr<Schema>& schema,
+                             ConflictPolicy conflict_resolution)
```

Comment:
Why not `policy` as above?

Reply:
vim language-server-client didn't rename all of them it seems.